tests/detectors/test_feature_detector.py

```diff
@@ -11,35 +11,40 @@
 from os.path import abspath
 
 from preggy import expect
-from tests.base import DetectorTestCase
+from tornado.testing import gen_test
 
+from tests.base import DetectorTestCase
 from thumbor.detectors.feature_detector import Detector as FeatureDetector
 
 
 class FeatureDetectorTestCase(DetectorTestCase):
-
-    def test_should_detect_multiple_points(self):
-        with open(abspath('./tests/fixtures/images/no_face.jpg'), 'rb') as f:
+    @gen_test
+    async def test_should_detect_multiple_points(self):
+        with open(abspath("./tests/fixtures/images/no_face.jpg"), "rb") as f:
             self.engine.load(f.read(), None)
 
-        FeatureDetector(self.context, 0, None).detect(lambda: None)
+        await FeatureDetector(self.context, 0, None).detect()
         detection_result = self.context.request.focal_points
         expect(len(detection_result)).to_be_greater_than(1)
-        expect(detection_result[0].origin).to_equal('alignment')
+        expect(detection_result[0].origin).to_equal("alignment")
 
-    def test_should_detect_a_single_point(self):
-        with open(abspath('./tests/fixtures/images/single_point.jpg'), 'rb') as f:
+    @gen_test
+    async def test_should_detect_a_single_point(self):
+        with open(
+            abspath("./tests/fixtures/images/single_point.jpg"), "rb"
+        ) as f:
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Variable name "f" doesn't conform to snake_case naming style](https://app.codacy.com/manual/heynemann/thumbor/pullRequest?prid=4967909)